src/math/Matrix4.js

```diff
@@ -822,9 +822,7 @@ class Matrix4 {
 
 	}
 
-	fromArray( array, offset ) {
-
-		if ( offset === undefined ) offset = 0;
+	fromArray( array = [], offset = 0 ) {
```

Comment:
The `array` parameter should never be optional when using `fromArray()`.

Reply:
Thanks for your review, I corrected it.

Reply:
@Mugen87 I do not think the array parameter should be optional when using `.toArray()` either. It is supposed to be populating an existing array, not instantiating one. Instantiating one violates the three.js policy.